app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -31,10 +31,14 @@ ListingController = ($scope, $state, $sce, $sanitize, SharedService, ListingServ
   $scope.isActiveTable = (table) ->
     $scope["active#{table}Class"] == 'active'
 
-  $scope.unitAreaRange = (units) ->
-    # TODO: actually find min/max
-    # if units.length == 1
-    units[0].Unit_Square_Footage
+  $scope.unitAreaRange = (summary) ->
```

Comment:
if you look here for example:
https://dahlia-qa-pr-137.herokuapp.com/api/v1/listings/a0X210000000IMM.json

It's really a `unitSummary`, a summary of all the units in the listing. Should probably rename it that for clarity... will do